guides/client/node.mdx

```diff
@@ -121,119 +101,145 @@ To run an instance of go-quai, you'll need to install a few dependencies. You ca
+
+**Global nodes** validate every shard in the network, i.e. Prime, all Region chains, and all Zone chains. Running a global node is only recommended for users who:
+
+- have more available computing power (strong CPU, more RAM, large SSD, etc.)
+- want to easily switch between shards for mining
+- want to provide chain data as an RPC service
 
-The `network.env` file houses a number of important parameters that we will configure to run our node correctly. The variables listed below will need to be set correctly for every experience level.
+**Slice nodes** validate a "slice" of all shards in the network. A slice is defined as the smallest set of shards a node can validate trustlessly, i.e. the Prime chain, any one Region chain, and any one Zone chain under the selected Region chain. A slice node validates only the shards in that slice, and runs lighter non-validation processes for all other shards in the network. Running a slice node is recommended for users who:
 
-- `COINBASE`: the address in each chain (location) that mining rewards are paid to. Note that there is one coinbase address per shard.
-- `NETWORK`: the network (testnet, devnet, etc.) your node is running on.
-- `SLICES`: the slices of the network that the node will run. Note that the default configuration is for a global node, and that this parameter must be modified to run a slice node.
-- `ENABLE_ARCHIVE`: whether or not the node will run in archive mode. Note that archive mode is required during the iron age testnet.
+- have less available computing power (less RAM, small SSD, etc.)
+- are not switching their miners between shards
+- are not providing chain data as an RPC service
+
+### Environment Variables
+
+After you've successfully installed go-quai, you'll need to configure your node. Go-quai uses a `config.toml` file to configure the node. The `config.toml` file is not located in the `go-quai` but rather in a directory at the root of your machine.
+
+The `config.toml` file houses a number of important parameters that we will configure to run our node correctly. The variables listed below will need to be set correctly for every experience level.
+
+- `environment`: the network (testnet, devnet, etc.) your node is running on.
+- `coinbases`: the addresses in each chain (location) that mining rewards are paid to. Note that you must have one address per slice you are running.
+- `slices`: the slices of the network that the node will run.
 
 <Note>This file also contains a number of more advanced parameters that _will not be covered_ in this article.</Note>
 
-Open the config file in your favorite text editor or IDE.
+<Steps>
+  <Step title="Open the Config File">
+    <Tabs>
+      <Tab title="Linux">
+        Open the `config.toml` file with your favorite text editor.
 
-- [vim](https://www.vim.org/)
-- [nano](https://www.nano-editor.org/)
-- [Visual Studio Code](https://visualstudio.microsoft.com/)
+        ```bash
```

Comment:
These instructions won't work until go-quai is run at least once, generating a default config.toml. Should include in the instructions an initial run of go-quai before changes are made.

We can also create a run command that just generates the config without starting the node.

Reply:
I would opt for a run command that generates the config. Running the node once, stopping it, configuring it, and then re-running it is a bit odd in terms of user flow. What would that command be?

Reply:
Yeah I agree but it doesn't exist we'd need to create it. I might be able to get it done this week but can't guarantee.

Reply:
Okay. Will keep this PR in review until you get a chance to make the command. Just keep me updated.

Reply:
Should be fixed in the most recent commit.